### ISSUE TITLE
Get model graph in JSON

### ISSUE THREAD

**Issue by Julia2505** [davidsandberg/facenet#1172]
Hello, David! Your repo is really usefull in my neural networks' studying. I have done softmax training on my own dataset. But to use model with identical structure it's neccessary to get graph in json format. Used model is this: 20180402-114759 (trained on VGGFace). Please, help me to get it or show the way to it...

**Comment by Nuha1412** [comment]
Do you know how to get the graph using this code ??
